packages/ng-query/src/lib/infinite-query.ts

```diff
@@ -45,8 +45,8 @@ type NgInfiniteQueryObserverReturnType<
     'subscribe'
   >;
 @Injectable({ providedIn: 'root' })
-class InfiniteQuery {
-  private instance = inject(QueryClient);
+class InfiniteQueryService {
```

Comment:
I'm still debating between the postfix - `Service` or `Provider`.

Reply:
I would choose `Provider` as postfix for the injection tokens and `Service` or `Impl` for the concrete implementations

Reply:
The injection tokens don't need a postfix. `UseQuery`, `UseInfiniteQuery`, and the rest, are consistent with the rest of the adapters.

Reply:
Voting for the `Service`

Reply:
If I understand your dilemma correctly, this class by its nature is a `Provider` (because it is used with a `new` in InjectionToken), but at the same time, if it's exposed to cover up #48, it would be better if it was `Service`.

Reply:
@Den-dp, exactly.